week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -12,12 +12,17 @@ let bottle = {
   volume: 0,
   fill: function() {
     // calling this function should make you bottles volume = 100;
+    return (this.volume = 100);
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    return (this.volume -= 10);
   },
   empty: function() {
     // this function should return true if your bottles volume = 0
+    if (this.volume == 0) {
+      return true;
+    }

```

Comment:
you can also instead simply write

```
if (!this.volume) {
   return true
}
```

or if you want to go even further, we can remove the `if` entirely.

```
empty: function() {
  return !this.volume
}
```

Can you see how these three are equivalent?